eng/common/templates/job/build-from-source.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+  jobNamePrefix: 'sourcebuild'    # Override for prefix for job names
+  managedOnly: false              # Specifies whether source is managed only. If so, one build platform is sufficient
+  portableOnly: false             # If true, builds one leg with portable flag enabled
+  prebuiltCheck: true             # Specifies whether to enforce prebuilt checking and fail if new prebuilts are introduced
+
+jobs:
+- template: /eng/common/templates/jobs/jobs.yml
+  parameters:
+    enablePublishUsingPipelines: true
+    enablePublishBuildArtifacts: true
+    enablePublishBuildAssets: true
+    artifacts:
+      publish:
+        artifacts: true
+        manifests: true
+    jobs:
+      # Only run fedora32 when either managedOnly or portableOnly is true.  Otherwise, run all legs.
+    - job: ${{ parameters.jobNamePrefix }}_fedora32
+      container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-32-20200512010618-163ed2a'
+      workspace:
+        clean: all
+      steps:
+      - template: ../steps/run-build-from-source.yml
+        parameters:
+          portableOnly: ${{ parameters.portableOnly }}
+
+    - job: ${{ parameters.jobNamePrefix }}_centos71
+      container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-3e800f1-20190501005343'
+      workspace:
+        clean: all
+      steps:
+      - template: ../steps/run-build-from-source.yml
+      condition: and( eq(${{ parameters.portableOnly }}, false), eq( ${{ parameters.managedOnly }}, false))
```

Comment:
Use template condition instead so the ones that don't run don't clutter the AzDO UI?